CRITICAL - 2022-03-16 08:47:27 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 40)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(40): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->home()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 08:55:08 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 40)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(40): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->home()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 08:55:33 --> file_get_contents(C:/xampp/htdocs/contratos/contratos/usuario.json): Failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 17)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(17): file_get_contents('C:/xampp/htdocs...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 08:56:48 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 08:57:31 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 08:57:38 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 09:19:55 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 09:25:00 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 09:29:45 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-16 09:29:57 --> file_get_contents(http://localhost:8080/api/pagamento): Failed to open stream: HTTP request failed! HTTP/1.1 401 

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp2\\htdoc...', 24)
#1 C:\xampp2\htdocs\contratos\app\Controllers\Home.php(24): file_get_contents('http://localhos...')
#2 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(825): App\Controllers\Home->pagina_inicial()
#3 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#4 C:\xampp2\htdocs\contratos\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp2\htdocs\contratos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
